Modules/clients/Controller/ClientslistController.php

```diff
@@ -207,6 +207,17 @@ public function editdeliveryaddressAction($id_space, $id){
             'formHtml' => $formd->getHtml($lang),
         ));
     }
+
+    /**
+     * Returns client's address
+     */
+    public function getAddressAction() {
+        $params = $this->request->params();
+        $id_client = $params["id"];
```

Comment:
why using params ? should not use global request params (at least the less possible)

just take input function parameters....